### ISSUE TITLE
KDB support

### ISSUE THREAD

**Issue by tcccapinfo** [BitcoinExchangeFH/BitcoinExchangeFH#142]
It seems like KDB is not yet supported, but I'm seeing some issues which indicate that there may have been some support in the past.

I'm thinking of adding KDB support myself, however, is there any KDB support out there that I can use / build off of, instead of writing all from scratch?

**Comment by gavincyi** [comment]
Yes apologies for the failed compatibility of kdb compared to the previous version while I'm implementing it in the coming versions very soon. Also open for contribution here. :)

I used qpython in the previous versions while AFAIK qpython is nothing magic but just a socket connection. So I'm open if you write it in low level socket for latency. I'm fine for either way. :)
